### PR TITLE
Histogram tool tips

### PR DESCRIPTION
**Pull request by a-mabe**
Added tool tip to histogram.js.

Added tool tip styling to histogram_style.css.

### CODE REVIEW

**Comment by ibnesayeed** [comment]
@a-mabe these changes look good to me, but commit messages are not helpful. Instead of telling which files were updated, it is generally more helpful to say what difference the change has made (e.g., `Clean unnecessary empty lines`).

Is it well-tested and ready to be merged?

File: public/scripts/histogram.js [hunk 1]
```diff
@@ -58,7 +63,20 @@ function getHistogram(fromYear, fromMonth, fromDate, toYear, toMonth, toDate, da
 	      .attr("transform", function(d) {
 			  return "translate(" + x(d.x0) + "," + y(d.length) + ")"; })
 	      .attr("width", function(d) { return x(d.x1) - x(d.x0) -1 ; })
-	      .attr("height", function(d) { return height - y(d.length); });
+	      .attr("height", function(d) { return height - y(d.length); })
+ 	      .on("mouseover", function(d) {    
+		    div .transition()    
+			.duration(200)    
+			.style("opacity", .8);    
+		    div .html("Number of Mementos: " + d.length + "<br/>" + d.x0.toString().substring(4,7) + " " + d.x0.toString().substring(11,15))
```
Comment (ibnesayeed) [inline public/scripts/histogram.js]: @a-mabe I am not too confident about `d.x0.toString().substring(4,7)` and `d.x0.toString().substring(11,15)`. What is in `d.x0` here? And how reliable is it when the number of mementos vary?

Comment (a-mabe) [reply public/scripts/histogram.js]: d.length gets the number of mementos for that specific bar, I've tested this with sites that have over 800 mementos and some with less than 20. d.x0 contains a variable of type object which holds the date for that bar. To extract the month and year from the object to be displayed, I've used .toString().substring(4,7) and .toString().substring(11,15).

**Comment by a-mabe** [comment]
Yes, it is well tested and ready. Thanks.

**Pull request merged**